ansible_collections/arista/avd/docs/how-to/first-project.md

```diff
@@ -94,27 +94,27 @@ AVD:
         AVD_L2LEAFS:
 ```
 
-In this file, you can also define management address of all devices. Because in this lab we use Jumphost, we will configure this address in the next section, but at least we configured on which port we will connect to eAPI using `ansible_port` variable.
+The management address of all devices can be defined in this file. Because a Jump-host will be used in this lab, we will configure the `ansible_port` variable to set the port number for eAPI connection, and the management address configuration is addressed in next section.
```

Comment:
super confusing that the example above is with `ansible_host` and this section mentioned `ansible_port`. And it is not covered in any later section

Reply:
I didn't pay attention. I just para-phrased it. Will fix it.